debian/postinst

```diff
@@ -67,8 +68,31 @@ upgrade_to_status_cache() {
     ua status 2>&1 > /dev/null
 }
 
+is_release_upgrade() {
+    # Return 0 if the package version we upgrade from does not match the
+    # current series (e.g. 14.04)
+    if test "${2#*$VERSION_ID}" = ${2}; then
```

Comment:
I don't think this line is going to work as intended, for a few reasons (I'm going to see if GH will let me do multiple comment threads). Consider:

```sh
#!/bin/sh

demo() {
    echo $2
}

demo
demo a
demo a b
```

I get:

```
$ ./foo.sh 1 2


b
```

As you can see, the function call means that `$2` is no longer referencing the arguments to the script, but the arguments to the function (whether or not there are any).

Reply:
fixed. Setting a global PREVIOUS_PKG_VER=$2 and using that throughout postinst for clarity/correctness